tox.ini

```diff
@@ -48,7 +48,7 @@ deps =
     numba
     pandas
 commands =
-    {posargs:pytest --cov --cov-report=term-missing -vv tests}
+    {posargs:pytest --cov --cov-report=term-missing -vv tests --ignore=./tests/scripts_test_result.py} 
```

Comment:
please explain in this pull request what this code does

Reply:
ignore scripts_test_result.py file when running tox